### PR TITLE
Moved demo dependencies to devDependencies

### PR DESCRIPTION
**Pull request by ColinEberhardt**
ref #17

### CODE REVIEW

File: demo/package.json [hunk 1]
```diff
@@ -2,19 +2,7 @@
...
-    "prismjs": "^1.16.0",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
-    "react-router-dom": "^5.0.0",
-    "react-scripts": "3.0.1",
-    "react-switch": "^5.0.0",
-    "typescript": "3.4.5",
-    "uuid": "^3.3.2"
-  },
+  "dependencies": {},
```
Comment (oriondean) [inline demo/package.json]: Shouldn't we keep actual runtime necessities as dependencies here? Typescript and react-scripts can move over to devDependencies but the rest should stay.

**Comment by maoo** [comment]
Thanks for this @ColinEberhardt ! Would be also possible to define `license: "Apache-2.0"` across all `package.json` files of the repo? TY!

**Comment by oriondean** [comment]
> Thanks for this @ColinEberhardt ! Would be also possible to define `license: "Apache-2.0"` across all `package.json` files of the repo? TY!

I believe this has been done by https://github.com/ScottLogic/openfin-react-hooks/pull/19 or have we missed a spot? :)

**Comment by maoo** [comment]
> > Thanks for this @ColinEberhardt ! Would be also possible to define `license: "Apache-2.0"` across all `package.json` files of the repo? TY!
> 
> I believe this has been done by #19 or have we missed a spot? :)

Yes! Sorry I missed the PR.

File: demo/package.json (hunk 1 above)
Comment (ColinEberhardt) [reply demo/package.json]: That's a good point @oriondean - the code isn't bundled, so those dependencies are required at runtime. I've updated and rebased.

**Pull request merged**